src/test/scala/io/sdkman/repos/VersionsRepoSpec.scala

```diff
@@ -199,6 +200,51 @@ class VersionsRepoSpec extends AnyWordSpec with Matchers with BeforeAndAfter wit
         }
       }
     }
+
+    "attempt to find all versions by platform where version starts with a major version and ends on a vendor suffix" in new TestRepo with TableDrivenPropertyChecks {
```

Comment:
How about we mix in the table driven property checks trait at the specification level. It feels like it should belong at the test level. wdyt?